tests/expectations/tests/bitfield-enum-basic.rs

```diff
@@ -68,8 +84,12 @@ impl ::std::ops::BitAndAssign for Buz {
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
 pub struct Buz(pub i8);
-pub const NS_FOO: _bindgen_ty_1 = _bindgen_ty_1(1);
-pub const NS_BAR: _bindgen_ty_1 = _bindgen_ty_1(2);
+impl _bindgen_ty_1 {
```

Comment:
Hmm, I didn't think about unnamed enums, but this is a problem... Because the constants are no longer in the top-level namespace, and their name is not deterministic.